Automattic/wp-calypso
# Assembler Level-Up: Use new patterns under feature flag

Related to https://github.com/Automattic/wp-calypso/issues/84893

## Proposed Changes

* Uses the feature flag `pattern-assembler/v2`
* We'll show all and only patterns from the new source site https://assemblerv2patterns.wordpress.com
  * No tags are needed for new patterns until they are moved to Dotcompatterns 
* Force using content from Assembler theme demo site to help test with existing sites

### Important notes 
- Patterns from dotcompatterns.wordpress.com that we want in Assembler v2 must be copied into the new source site
- Page patterns also have to be copied into the new source site and tagged with the tag `Assembler_page`

### Assembler on Calypso Live

https://github.com/Automattic/wp-calypso/assets/1881481/acaeed57-b20c-4b3d-9f8d-9bb4b8e16346



## Testing Instructions


* Sandbox D131496-code
* Access the Assembler using the Calypso Live link below
  * Click "Calypso Live (direct link)" and then create a new site, then continue until the Design picker and click "Design your own" to access the assembler.
  * Another option is using an existing site, click the same link, and then update the URL to be `/setup/with-theme-assembler/patternAssembler?siteSlug=YOUR-SITE-URL`
* Add the param `flags=pattern-assembler/v2` to the URL
* Verify you see only the patterns from assemblerv2patterns.wordpress.com
* Add some patterns
* Verify there are blank patterns because they contain a reference to a `wp_block` pattern
* Verify the styles and fonts work as expected - The variations are still fetched from Creatio 2
* Verify pages are still not available
* Continue until the editor and verify the Home template is created as expected




## Pre-merge Checklist
